src/api/keyword/asyncEditSubkeyword.js

```diff
@@ -0,0 +1,17 @@
+import fetchHandler from "..";
+import { BASE_URL } from "../../config/constants";
+
+const asyncEditSubkeyword = async (subkeywordInfo, keywordId) => {
+  const fetchInfo = {
+    url: `${BASE_URL}/keywords/${keywordId}`,
+    method: "POST",
```

Comment:
```suggestion
    method: "PUT",
```
[API 명세](https://www.notion.so/13154df96043808196b4fcc2678ad2cf)에 따라 `PUT`으로 수정 예정입니다!